core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -54,7 +54,8 @@
     private DataSize maxLocalExchangeBufferSize = DataSize.of(32, Unit.MEGABYTE);
     private DataSize maxIndexMemoryUsage = DataSize.of(64, Unit.MEGABYTE);
     private boolean shareIndexLoading;
-    private int maxWorkerThreads = Runtime.getRuntime().availableProcessors() * 2;
+    private double maxWorkerThreadsProcessorMultiplier = 2.0;
```

Comment:
Maybe instead it should be a String with a custom syntax supporting either: 16 (absolute value), C16 (16 * core count) similiar to Maven parallelism settings

Reply:
Or even a custom type: `ThreadCount` that does the validation, parsing etc

Reply:
It's good to know the actual number which is printed out during system startup in logs